services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/Utils.kt

```diff
@@ -13,6 +13,7 @@ private val logger = KotlinLogging.logger {}
 
 private val businessIdRegex = "^(\\d{7})-(\\d)\$".toRegex()
 private val businessIdMultipliers = listOf(7, 9, 10, 5, 8, 4, 2)
+private val ovtRegex = "^0037(\\d{8})[|\\w]{0,5}\$".toRegex()
```

Comment:
Why are `|` and `_` allowed but no other punctuation? Should the identifying part also be digits? Is there a source for the definition?

Reply:
Looks like ProCounter only accepts numbers as the OVT.

https://support.procountor.fi/hc/fi/articles/360000256417-Laskuaineiston-siirtotiedosto
Laskutietue -> Nro. 38

Reply:
That sounds more accurate than the one I used (https://support.maventa.fi/fi/support/solutions/articles/77000468510-mik%C3%A4-on-ovt-tunnus-verkkolaskuoperaattori-ja-verkkolaskuosoite-) which only talks about "characters" in ovt.